Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'jira-ruby', :group => :development
 
 group :test do
   gem 'rspec'
-  gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
+  gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.12.0')
```

Comment:
Hmmm, is that what's actually being used for the platform side, though?  This recent job, for example, appears to be picking up 2.12.0:

https://jenkins.puppetlabs.com/view/All%20in%20One%20Agent/view/Master/view/Puppet/job/platform_aio-puppet_intn-sys_master/75/SLAVE_LABEL=beaker,TEST_TARGET=redhat-7-x86_64/consoleFull